### DISCUSSION TITLE
Custom Blog pages not working 404 error

### DISCUSSION THREAD

**Discussion by UnaFordable** [community/community#148119] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I purchased the GitHub for dummies (2nd Edition) by Sarah Guthals to better understand GitHub. When I wrote my code to display my blogs, I get a 404 error. It seems no matter what themes I use or what files I add, I can never get my blogs to display.

![IMG_2778](https://github.com/user-attachments/assets/89a2afed-8ef1-4d21-b749-0c926829984e)
![IMG_2779](https://github.com/user-attachments/assets/1673273f-9a9b-47f8-bca3-c741e96a44e1)

Here I my page: https://unafordable.github.io/UnaFordable/
Here is my code: https://github.com/UnaFordable/UnaFordable

**Comment by mecodeatlas** [comment]
Thanks for your post, @UnaFordable! We worked with the Pages team to create a [guide](https://github.com/orgs/community/discussions/61073) to help you troubleshoot this error. Check out the [Pages 404 error guide](https://github.com/orgs/community/discussions/61073) for assistance. And don't forget to spread the knowledge 💡 and the guide to your fellow community members.
